pylint/checkers/imports.py

```diff
@@ -448,25 +446,7 @@ def _compute_site_packages():
         def _normalized_path(path):
             return os.path.normcase(os.path.abspath(path))
 
-        paths = set()
-        real_prefix = getattr(sys, "real_prefix", None)
-        for prefix in filter(None, (real_prefix, sys.prefix)):
-            path = sysconfig.get_python_lib(prefix=prefix)
-            path = _normalized_path(path)
-            paths.add(path)
-
-        # Handle Debian's derivatives /usr/local.
-        if os.path.isfile("/etc/debian_version"):
-            for prefix in filter(None, (real_prefix, sys.prefix)):
-                libpython = os.path.join(
-                    prefix,
-                    "local",
-                    "lib",
-                    "python" + sysconfig.get_python_version(),
-                    "dist-packages",
-                )
-                paths.add(libpython)
-        return paths
+        return {_normalized_path(path) for path in astroid.modutils.STD_LIB_DIRS}
```

Comment:
Could you add typing to this method? 😏

Reply:
Sure thing. The normalization is probably already done in astroid, but I didn't audit it over there, so I was reluctant to change it here.

Reply:
I don't think it actually is. At least `normcase` might be necessary/new here.